web/client/src/library/components/editor/EditorCode.tsx

```diff
@@ -191,23 +191,65 @@ function CodeEditorDefault({
   }, [dialect])
 
   useEffect(() => {
-    setValue(content)
-  }, [content])
+    updateEditor(content)
+  }, [editorView, content, extensionsAll])
+
+  function updateEditor(content: string = ''): void {
+    if (isNil(editorView)) return
+
+    const state = editorView.state
+    const text = state.doc.toString()
+
+    if (text === content || isNil(diffChars)) return
```

Comment:
What's the relationship between text and content?

Reply:
text - is current editor's content, I'll rename